Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/BufferDescriptor.h

```diff
@@ -67,8 +67,6 @@ namespace AZ
 
         BufferBindFlags GetBufferBindFlags(ScopeAttachmentUsage usage, ScopeAttachmentAccess access);
 
-        AZ_ASSERT_NO_ALIGNMENT_PADDING_BEGIN
```

Comment:
Why are we removing these? I couldnt find an explanation anywhere.

Reply:
The warning has been enabled across O3DE, this macro is no longer needed (the macro was enabling the warning for this specific block)